Trappist/src/Promact.Trappist.Utility/GlobalUtil/Util.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Promact.Trappist.Utility.Constants;
+using System;
+using System.Linq;
+
+namespace Promact.Trappist.Utility.GlobalUtil
+{
+    public class Util : IUtil
+    {
+        private static Random random = new Random();
+        private readonly IStringConstants _stringConstants;
+        public Util(IStringConstants stringConstants)
+        {
+            _stringConstants = stringConstants;
+        }
+        public string RandomLinkGenerator(int length)
```

Comment:
change its name to GenerateRandomString as it is not specific to only link generation